js/models/conversations.js

```diff
@@ -655,6 +671,7 @@
         await this.respondToAllPendingFriendRequests({
           response: 'accepted',
           direction: 'incoming',
+          status: ['pending', 'expired'],
```

Comment:
I think you also wanna make this `respondToAllFriendRequests`

Reply:
Good catch!